project/Dependencies.scala

```diff
@@ -212,6 +212,13 @@ object Dependencies {
     )
   )
 
+  val PostgreSqlCdc = Seq(
+    libraryDependencies ++= Seq(
+      "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.0", // BSD-style,
```

Comment:
`scala-java8-compat` comes as a transitive dependency from `akka-actor`. Therefore we should not set it as a dependency of this connector, because of a possible version mismatch.

Reply:
thanks. i'll take it out.